AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -101,6 +125,58 @@ public final class PayToComponent: PaymentComponent,
         return item
     }()
 
+    /// The  account holder firstname text input item.
+    internal lazy var firstNameInputItem: FormTextInputItem = {
+        let item = FormTextInputItem(style: configuration.style.textField)
+        // TODO: Add translation
+        item.title = localizedString(LocalizationKey(key: "Account holder first name"), configuration.localizationParameters)
+        item.placeholder = localizedString(LocalizationKey(key: "Account holder first name"), configuration.localizationParameters)
+        item.identifier = ViewIdentifierBuilder.build(
+            scopeInstance: self,
+            postfix: ViewIdentifier.firstNameInputItem
+        )
+        return item
+    }()
+
+    /// The  account holder lastname text input item.
+    internal lazy var lastNameInputItem: FormTextInputItem = {
+        let item = FormTextInputItem(style: configuration.style.textField)
+        // TODO: Add translation
+        item.title = localizedString(LocalizationKey(key: "Account holder last name"), configuration.localizationParameters)
+        item.placeholder = localizedString(LocalizationKey(key: "Account holder last name"), configuration.localizationParameters)
+        item.identifier = ViewIdentifierBuilder.build(
+            scopeInstance: self,
+            postfix: ViewIdentifier.lastNameInputItem
+        )
+        return item
+    }()
+
+    /// The identifier picker item.
+    internal lazy var identifierPickerItem: FormIdentifierPickerItem = {
+        let selectableValues = AccountIdentifier.allCases.map { account in
+            BasePickerElement(
```

Comment:
I see both `FormIdentifierPickerElement` and `BasePickerElement` and it's confusing. Shouldn't there be only one of them? In this case, the new one `FormIdentifierPickerElement`

Reply:
Modified the code :)